pyproject.toml

```diff
@@ -49,15 +50,13 @@ url = "https://download.pytorch.org/whl/cu118"
 priority = "primary"
 
 [tool.poetry.extras]
-dev = [
+base = [
```

Comment:
Why base instead of dev? These are development packages, rather than required for running the code "in production", no?

Reply:
the dev is still there, see below